modules/module-info.php

```diff
@@ -932,3 +932,10 @@ function jetpack_more_info_copy_post() {
 	esc_html_e( 'Create a new post based on an existing post.', 'jetpack' );
 }
 add_action( 'jetpack_module_more_info_copy-post', 'jetpack_more_info_copy_post' );
+
+function jetpack_private_more_info() {
+	esc_html_e(
+		'Make your site private. It will only be visible to registered users.'
+	);
+}
+add_action( 'jetpack_module_more_info_private', 'jetpack_private_more_info' );
```

Comment:
We would need a link to a support doc here as well, using the other filter used for the other modules above.

Reply:
@jeherve I'm not sure I understood this correctly. Are you suggesting to create a new support doc for Private Sites at jetpack.com/support and then link it here?

Reply:
Yes, exactly.

Reply:
Does it make sense to do this after we merge this PR?

Reply:
Added the link to the support doc in [1cc8cc6](https://github.com/Automattic/jetpack/pull/10333/commits/1cc8cc6370f31f6370d4eab3776374cd8db2bca3).